Gemfile

```diff
@@ -14,6 +14,7 @@ gem "puma", '>= 3.12.0'
 gem "rack", '>= 2.0.5'
 gem "rails-auth", '>= 2.1.4'
 gem "sinatra", '>= 2.0.4'
+gem "puppet", github: 'beechtom/puppet', branch: 'GH-1403-hiera-lookup'
```

Comment:
#### TODO
Remove the commit that makes this change when/if the changes are made to pal.